src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -145,10 +145,12 @@ protected override void SetupNewTextView(IVsTextView textView)
                         var snapshot = subjectBuffer.CurrentSnapshot;
                         var fullSpan = new SnapshotSpan(snapshot, start: 0, length: snapshot.Length);
                         var tagger = outliningTaggerProvider.CreateTagger<IOutliningRegionTag>(subjectBuffer);
+                        using (var disposable = tagger as IDisposable)
```

Comment:
I'd be OK with a hard cast here -- if this doesn't implement IDisposable we've got problems.